packages/api/schema/stryker-core.json

```diff
@@ -204,6 +204,47 @@
             "type": "string"
           },
           "default": []
+        },
+        "specificMutants": {
```

Comment:
The instrumentor gets passed the `InstrumenterOptions` section of `StrykerOptions`, rather than the whole thing. I thought it would be better to have the options validator take the data from the `mutate` property and transform it into a well-defined object that gets passed into the instrumentor rather than change it so that the instrumentor gets passed the entire `StrykerOptions` object just so that the instrumtor can try and parse it out of `mutate`.

Reply:
Sorry, but I would have to disagree with adding `specificMutants` here. Adding it here would mean users can provide specific mutants both in `--mutate` as well as here. This increases our API surface, maintenance, documentation, etc.

I looked into the code, but it seems that the `InstrumenterOptions` are already separate from the `MutatorDescriptor`. In other words, you could provide it to the instrumentor with something like this:

```js
// Instrument files in-memory
const instrumentResult = await instrumenter.instrument(this.inputFiles.filesToMutate, {
  ...this.options.mutator,
  mutationRange: this.inputFiles.mutationRange
});
```

(I think `mutationRange` might be a better word in this context, leaving "specificMutants" to be used as an array of id's in the future).

It would mean adding the `mutationRange` to the `InputFileCollection`, which kind of makes sense.

We could also think a bit more outside the box and specify a class that inherits from `File` the `mutationRange`, but that would mean doing something with object-orientation, which we've kind of moved away from in recent years.